index.html

```diff
@@ -0,0 +1,147 @@
+        <h3>Gaphic Design</h3>
+        <p>We offer comprehensive services in web development, graphic design, and digital marketing. As junior web developers, we create dynamic, user-friendly websites, design captivating graphics, and implement effective digital marketing strategies to help your business thrive online.</p>
+        <a href="#" class="btn">Read More</a>
+      </div>
+      <div class="services-box">
+        <i class='bx bx-code-alt'></i>
+        <h3>Web Development</h3>
+        <p>We offer comprehensive services in web development, graphic design, and digital marketing. As junior web developers, we create dynamic, user-friendly websites, design captivating graphics, and implement effective digital marketing strategies to help your business thrive online."</p>
+        <a href="#" class="btn">Read More</a>
+      </div>
+      <div class="services-box">
+        <i class='bx bx-bar-chart-alt-2'></i>
+        <h3>Digital Marketing</h3>
+        <p>We offer comprehensive services in web development, graphic design, and digital marketing. As junior web developers, we create dynamic, user-friendly websites, design captivating graphics, and implement effective digital marketing strategies to help your business thrive online."</p>
+        <a href="#" class="btn">Read More</a>
+      </div>
+    </div>
+  </section>
+  <!-- portfolio section design -->
+  <section class="portfolio" id="portfolio">
+    <h2 class="heading">Latest <span>Project</span></h2>
+    <div class="portfolio-container">
+      <div class="portfolio-box">
+        <img src="img/atribute-page.png" alt="">
+        <div class="portfolio-layer">
+          <h4>Web Design</h4>
+          <p>tribute-page dedicated to Ahmad-Shah-Baba showcasing his life and achievements.
+          </p>
+          <a href="#"><i class='bx bx-link-external'></i></a>
+        </div>
+      </div>
+      <div class="portfolio-box">
+        <img src="img/img.png.png" alt="">
+        <div class="portfolio-layer">
+          <h4>Web Design</h4>
+          <p>comprehensive skincare page product reviews,and personalized routines. The page is designed to be user-friendly providing valuable information to help users achieve their best skin</p>
+          <a href="#"><i class='bx bx-link-external'></i></a>
+        </div>
+      </div>
+      <div class="portfolio-box">
+        <img src="img/Screenshot (14).png" alt="">
+        <div class="portfolio-layer">
+          <h4>Web Design</h4>
+          <p>Simple Survey Form Lorem ipsum dolor sit.</p>
+          <a href="#"><i class='bx bx-link-external'></i></a>
+        </div>
+      </div>
+      <div class="portfolio-box">
+        <img src="img/Screenshot (5).png" alt="">
+        <div class="portfolio-layer">
+          <h4>Web Design</h4>
+          <p>Coffee Shop web Page that help people order and reserve the place for thier self</p>
+          <a href="#"><i class='bx bx-link-external'></i></a>
+        </div>
+      </div>
```

Comment:
![Screenshot 2024-06-06 124436](https://github.com/Saqibi4213/my-portfolio/assets/96838030/1a5fc11e-e0dd-4302-b124-67059086680c)

- Try to add the Source Code (Your project github link) besides your Demo (Live link).